test/libyul/yulOptimizerTests/rematerialiser/for_continue_with_assignment_in_post.yul

```diff
@@ -0,0 +1,54 @@
+{
+	let a
+	let b
+	let c
+	for {
+		let i := 0
+		b := origin()
+		c := origin()
+	}
+	lt(i, 10)
+	{
+		i := add(a, b)
+		b := 0x42
+		c := 0x90
+	}
+	{
+		a := origin()
+
+		b := caller()
+		if callvalue() { continue }
+		a := caller()
+	}
+	let x := b // does not rematerialize as b may be either origin() or 0x42 (btw: not caller())
+	let y := c // does not rematerialize as c may be either origin() or 0x90
+}
+// ----
+// rematerialiser
+// {
+//     let a
+//     let b
+//     let c
+//     for {
+//         let i := 0
+//         b := origin()
+//         c := origin()
+//     }
+//     lt(i, 10)
+//     {
+//         i := add(a, caller())
+//         b := 0x42
+//         c := 0x90
```

Comment:
Ok, final comment: Please do not use constants, they might be more expensive than e.g. `caller()` and thus not rematerialize.

Reply:
didn't know constants are more expensive. I just didn't wanna use caller() everywhere. :)

Reply:
Look for a cost of `BASE` in `Instruction.cpp`

Reply:
I've replaced the constants with other caller() / callvalue().